crates/rustc_codegen_spirv/src/attr.rs

```diff
@@ -298,6 +297,31 @@ impl CheckSpirvAttrVisitor<'_> {
                                 span,
                                 "attribute is only valid on a parameter of an entry-point function",
                             );
+                        } else {
+                            // FIXME(eddyb) should we just remove all 5 of these storage class
+                            // attributes, instead of disallowing them here?
```

Comment:
Would there ever be a need to use these attributes in `spirv-std`? If not, I agree that we should remove them.